tests/component/test_task_events.py

```diff
@@ -229,9 +229,7 @@ def test___ai_task____run_multiple_finite_acquisitions_with_varying_every_n_samp
     ] == every_n_samples_event_counts
 
 
-@pytest.mark.grpc_xfail(
-    reason="AB#2395958: GrpcStubInterpreter does not report event registration errors to caller"
-)
+@pytest.mark.grpc_xfail(reason="Requires NI gRPC Device Server version 2.2 or later")
```

Comment:
@zhindes I don't want to remove the xfail until 2.2 is closer to release, since we will probably continue to test with 2.1.